CHANGELOG.rst

```diff
@@ -8,6 +8,31 @@ GitHub issues are referenced, and can be viewed with hyperlinks on the `github r
 
 .. _`github releases page`: https://github.com/Libensemble/libensemble/releases
 
+Release 0.9.3
+-------------
+
+:Date: October 12, 2022
+
+New capabilities:
+
+* Persistent functions can send intermediate values without the Manager assuming the worker is ready to receive. #880
```

Comment:
Should this be "Persistent functions can be sent intermediate values without the Manager assuming the worker is ready to receive them"?

Reply:
Even that doesn't seem right....

Reply:
This refers to the `stay_active` argument in `PersistentSupport.send()`, so I think it should more refer to what the user does, rather than internally how the argument is handled.

Reply:
New persistent support function to cancel sim_ids (``request_cancel_sim_ids``) and a ``keep_state`` option for persistent worker send, which if True, will not update ready_to_receive state of the worker.

Reply:
Two bullets:

- New persistent support function to cancel sim_ids (request_cancel_sim_ids) 
- keep_state option for persistent worker send, which if True, will not update ready_to_receive state of the worker.

Reply:
New persistent support function to cancel `sim_ids` (`request_cancel_sim_ids`). 
A new `keep_state` option for persistent workers: this lets the manager know
that the information being sent is intermediate.